chapter07/src/test/scala/example/Chapter07_01_examplesSpec.scala

```diff
@@ -4,15 +4,18 @@ import cats.kernel.Semigroup
 import cats.{Functor, Monoid, derive}
```

Comment:
Get rid of this and Flattenable, it's not from this chapter ;)

Reply:
i didnt write that though lol. that's just brought in from merging from upstream, which is your repo

Reply:
Ah maybe you removed it in yours, and since I didn't rebase, it's like this.